build/bootstrap/refresh-autoshutdown.cron.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+autoshutdown_script=/root/autoshutdown.cron.sh
+autoshutdown_script_url=https://raw.githubusercontent.com/cockroachdb/cockroach/master/build/bootstrap/autoshutdown.cron.sh
+
+if [[ ! -e $autoshutdown_script ]]; then
+  echo "skipping update, still bootstrapping?"
```

Comment:
Is there a reason to have this condition? If the file is missing, do we not want to just create it anyway?

Reply:
This is to prevent a race condition, when still bootstrapping. This script is ran as a "startup-script" (set by `gceworker.sh create`), so we can skip running it before the machine is bootsrapped.
It's not a big deal if we remove this check, because the autoshutdown script is not yet scheduled to run by cron.